metrics/web/lib/common/presentation/navigation/models/page_parameters_model.dart

```diff
@@ -0,0 +1,10 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:equatable/equatable.dart';
+
+/// An abstract class, that represents the application page parameters.
```

Comment:
:pencil:
https://developers.google.com/tech-writing/one/short-sentences#distinguish_that_from_which
> Place a comma before **which**; do not place a comma before **that**.

```suggestion
/// An abstract class that represents the application page parameters.
```

Please apply to all doc in this PR where necessary.